pca_tools/check_lists.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+lists = []
+all_missing = []
+for file in os.listdir("."):
+    if file.endswith(".list"):
+	lists.append( file )
+
+lists.sort()
+
+for list in lists:
+	missing = []
+	with open(list) as f:
+    		content = f.readlines()
+		content = [x.strip() for x in content] 
+		max_run = max(content)
+		min_run = min(content)
+		
+		# check whether the file is stored at feyman
+		path = "/lustre/scratch/epp/neutrino/snoplus/TELLIE/TELLIE_PCA_RUNS"
+		
+		for run in content:
+			file_string = "SNOP_0000" + run + "_000.zdab"
```

Comment:
Not sure this is the default filename, but either way it should not be hardcoded this way.

Reply:
modified to string formating